### PR TITLE
Add model to mu-cl-resource configs

### PR DESCRIPTION
**Pull request by abeforgit**
Model docs here: https://github.com/brechtvdv/model-templates
Also bumps mu-cl-resources since we need inheritance here

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ services:
     restart: always
     logging: *default-logging
   resource:
-    image: semtech/mu-cl-resources:1.20.0
+    image: semtech/mu-cl-resources:latest
```
Comment (claire-lovisa) [inline docker-compose.yml:64]: Why not use a fixed version ? I'm always a bit scared of `latest`, we don't really have control of what goes there.
Plus I just tested and with latest mu-cl-resource breaks on my stack, but v1.20.0 seems to be working as expected

File: config/resources/domain.json [hunk 2]
```diff
@@ -6,9 +6,146 @@
...
+        "reason": {
+          "type": "string",
+          "predicate": "ext:reason"
+        }
+      },
+      "relationships": {
+        "concept": {
+          "predicate": "ext:concept",
+          "target": "concept",
+          "cardinality": "one"
```
Comment (claire-lovisa) [inline config/resources/domain.json:132]: Should it be `"many"` ?

Comment (abeforgit) [reply config/resources/domain.json:132]: ah yea according to the model I guess it does
bit weird, how would they combine semantically? as an `or` I suppose?

Comment (claire-lovisa) [reply config/resources/domain.json]: Yeah I also found it strange but :woman_shrugging:

**Pull request merged**